src/Homely.Storage.Blobs/CacheControlType.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Homely.Storage.Blobs
+{
+    public enum CacheControlType
+    {
+        None = 0,
+        NoStore,
+        NoCache
+    }
+
+    public static class CacheControlTypeHelper
+    {
+        public static string CacheControlToString(CacheControlType cacheControlType)
+        {
+            switch (cacheControlType)
+            {
+                case CacheControlType.NoStore:
+                    return "no-store";
+                case CacheControlType.NoCache:
+                    return "no-cache";
+                default:
+                    return string.Empty;
+            }
+        }
+
+        public static CacheControlType GetCacheControlType(string stringValue)
+        {
+            if (stringValue.Equals("no-store", StringComparison.OrdinalIgnoreCase))

```

Comment:
what happens if `stringValue` is null/empty? will this crash?

Reply:
good point! empty string won't crash but null will.. Fixed that up now.